### PR TITLE
update nasal-nasal mappings

### PR DESCRIPTION
**Pull request by LinguList**
@tresoldi, please check what I did. It now accepts `nm` as a cluster.

### CODE REVIEW

**Comment by tresoldi** [comment]
:+1:

**Comment by tresoldi** [comment]
Requesting @xrotwang approval so we can merge and move forward.

**Review by xrotwang — APPROVED** [review]

**Comment by LinguList** [comment]
NO, please do not merge this. These changes are dangerious, as they have not been tested on CLTs.

File: src/pyclts/transcriptionsystem.py [hunk 1]
```diff
@@ -207,7 +207,9 @@ def _parse(self, string):
                         string, sound1, sound2, self)
                 elif sound1.type == 'consonant' and \
                         sound1.manner in ('stop', 'implosive', 'click', 'nasal') and \
-                        sound2.manner in ('stop', 'implosive', 'affricate', 'fricative'):
+                        sound2.manner in (
```
Comment (LinguList) [inline src/pyclts/transcriptionsystem.py:210]: YOu have not verified the results on the whole of PYclts data, the modifications you propose are intrusive to the code and wil result in tons of additional clusters. I therefore close the PR.

**Pull request closed without merging**